projects/ngx-charts-on-fhir/src/public-api.ts

```diff
@@ -34,3 +34,4 @@ export * from './lib/fhir-mappers/observation/simple-observation-mapper.service'
 export * from './lib/fhir-mappers/medication-request/simple-medication-mapper.service';
 export * from './lib/fhir-mappers/medication-request/duration-medication-mapper.service';
 export * from './lib/utils';
+export * from './lib/datasets/blood-pressure-mapper';
```

Comment:
blood-pressure-mapper should be in the FhirMappersModule with all the other mappers. Put it in the `./lib/fhir-mappers/observation/` dir